governance/program/src/processor/process_create_realm.rs

```diff
@@ -57,11 +61,16 @@ pub fn process_create_realm(
         rent,
     )?;
 
-    let council_token_mint_address = if let Ok(council_token_mint_info) =
-        next_account_info(account_info_iter)
-    // 7
-    {
-        let council_token_holding_info = next_account_info(account_info_iter)?; //8
+    let realm_custodian = if config_args.use_custodian {
```

Comment:
How does this work if `council_token_mint = Some(...)` and `realm_custodian = None`? Is that the situation that's meant to fail?

Reply:
No, it was not meant to fail, but  since the custodian was removed in the next PR it's irrelevant now.